frontend/package.json

```diff
@@ -34,7 +34,7 @@
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "craco start",
+    "start": "craco --openssl-legacy-provider start",
```

Comment:
Dont do this. Instead, make sure youre using npm 16.

Reply:
Fixed